docs/ff-concepts/adding-customization/cloud-functions.md

```diff
@@ -281,6 +281,30 @@ If you encounter deployment errors, it may be helpful to check out [this communi
 
 </details>
 
+<details>
+<summary>I am getting "Error: Unable to initialize Storyboard</summary>
+<p>
+This error typically occurs because the initial page has n
```

Comment:
I guess this is added by mistake?

Reply:
Ya, it was by mistake! removed it now. Thanks!